content/docs/studio/create-view.md

```diff
@@ -0,0 +1,54 @@
+# Create a View
+
+To create a new view for your repository, follow these steps.
+
+1. Click on `Add a View`. All the organizations that you have access to will be
+   listed. ![](/img/studio/create_view.png)
+
+2. Open the organization whose repository you want to connect to. You can also
+   use the search bar to search directly for the repo which you want to connect
+   to. ![](/img/studio/select_repo.png)
+
+3. Select the Git repository for which you want to create a view. For this,
+   hover over the required repository and click on `Connect`.
```

Comment:
Maybe minor: Looks like 2 and 3 are the same step. Could be combined I think.

Reply:
> Also, it looks like you can click anywhere, doesn't have to be in the `Connect` button.

Reply:
> Looks like 2 and 3 are the same step. Could be combined I think.

They are slightly different - 2 is about finding the repo and 3 is about connecting to it. Not much difference, but I think we can let it be.

> doesn't have to be in the Connect button.

Hmm, ok. But maybe in the docs, we will say click on Connect, just for clarity.

Reply:
UX-wise they seem like the same step to me.
Saying "Click on `Connect`" doesn't clarify IMO, since you can click anywhere.
Lets please leave unresolved discussions unresolved 👍

Reply:
sure 👍

Reply:
Added to #2468